src/Initialization/init_JuMP.jl

```diff
@@ -1,34 +1,34 @@
-using JuMP.MathOptInterface: AbstractOptimizer, OptimizerWithAttributes
+using JuMP.MOI: AbstractOptimizer, OptimizerWithAttributes
```

Comment:
If you need these, you should probably instead declare a `Pkg` dependency on `MathOptInterface` explicitly? It's weird to import structs/methods from a dependency of a package.

Reply:
thanks, done in https://github.com/JuliaReach/LazySets.jl/pull/3245#pullrequestreview-1346354090